.github/workflows/auto-reviewer.yml

```diff
@@ -0,0 +1,13 @@
+name: "Reviewer lottery"
+on:
+  pull_request:
+    types: [opened, ready_for_review, reopened]
+    tags-ignore:
+    - pair-programmed
```

Comment:
nitpick (non-blocking): This looks slightly misindented, I imagine Prettier will want to reformat it